RevenueCat/Scripts/Package.cs

```diff
@@ -1,4 +1,6 @@
+using System;
```

Comment:
is this needed?

Reply:
It _was_ at one point but might not be anymore 😛 I'll remove it not!

Reply:
Ah yeah, this is needed for `Obsolete` 🤷‍♂️